### ISSUE TITLE
[Help] How to train using a custom dataset (different labels) 

### ISSUE THREAD

**Issue by breznak** [open-mmlab/mmsegmentation#2134]
Hello, I am new to mmsegmentation, 

my goal is to train a (from scratch) model on a custom dataset with different labels from supported datasets (eg. labes are ['a','b']). 

I have read the tutorial on "Custom dataset" https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/tutorials/customize_datasets.md 

but it only covers how to convert your data structure into a supported format (eg. cityscapes, VOC, ...). 

But I need to train using a custom dataset (I can use eg the cityscapes format) but I need to use different labels. 
mmdetection shows how to support that: 
https://mmdetection.readthedocs.io/en/latest/tutorials/customize_dataset.html 
Basically you need to change the `num_classes=` of output neurons in the output head. 

Please let me know how is it possible for mmsegmentation tasks? 
I have looked in the configs (dataset `cityscapes.py`, model eg `configs/pspnet/pspnet_r18-d8_512x1024_80k_cityscapes.py`) but I don't see how to change what labels get recognized? 

(I use CVAT to annotate the data and can export to most of the formats)

 
Thank you for advice,

**Comment by xiexinch** [comment]
Hi @breznak,
In most cases, you just need to inherit [CustomDataset](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py), then set `CLASSES` and `PALETTE`, and finally write the dataset config file.

For example:

https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/coco_stuff.py

https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/datasets/coco-stuff10k.py
